apps/docs/pages/guides/auth/managing-user-data.mdx

```diff
@@ -224,8 +224,8 @@ language plpgsql
 security definer set search_path = public
 as $$
 begin
-  insert into public.profiles (id)
-  values (new.id);
+  insert into public.profiles (id, first_name, age)
+  values (new.id, new.raw_user_meta_data['first_name']#>>'{}', cast(new.raw_user_meta_data['age'] as int));
```

Comment:
```suggestion
  values (new.id, new.raw_user_meta_data ->> 'first_name', new.raw_user_meta_data['age']::integer);
```

I haven't tested this but I think this might be a bit clearer at expressing what the code is trying to do